include/odp/api/spec/init.h

```diff
@@ -172,6 +198,12 @@ typedef struct odp_init_t {
 	 */
 	odp_feature_t not_used;
 
+	/** Application memory model. The main application thread has to call
+	 *  odp_init_global() and odp_init_local() before creating threads that
+	 *  share ODP data.
+	 */
+	odp_mem_model_t mem_model;
```

Comment:
The default value needs to be defined: "The default value is ODP_MEM_MODEL_THREAD"